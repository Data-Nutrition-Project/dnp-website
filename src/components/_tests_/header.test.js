import React from "react"
import renderer from "react-test-renderer"
import Header from "../Header/index"

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Header />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
