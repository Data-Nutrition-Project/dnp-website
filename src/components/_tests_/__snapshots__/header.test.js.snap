// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="header"
>
  <div
    className="headerWrap"
  >
    <div
      className="container clearfix"
    >
      <div
        className="primaryMenuTrigger"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <i
          className="icon-reorder"
        />
      </div>
      <div
        className="logo"
      >
        <a
          aria-current="page"
          className="standardLogo"
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          style={Object {}}
        >
          <img
            alt="logo"
            className="logoImg"
            src="logo-s.png"
          />
        </a>
      </div>
      <nav
        className="primaryMenu"
      >
        <div
          className="justify-content-end primaryNav nav"
          onKeyDown={[Function]}
        >
          <div
            className="nav-item"
          >
            <a
              className="navLink"
              href="/#section-mission"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              <div>
                Mission
              </div>
            </a>
          </div>
          <div
            className="nav-item"
          >
            <a
              className="navLink"
              href="/#section-problem"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              <div>
                Problem
              </div>
            </a>
          </div>
          <div
            className="nav-item"
          >
            <a
              className="navLink"
              href="/#section-solution"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              <div>
                Solutions
              </div>
            </a>
          </div>
          <div
            className="nav-item"
          >
            <a
              className="navLink"
              href="/#section-team"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              <div>
                Team
              </div>
            </a>
          </div>
          <div
            className="nav-item"
          >
            <a
              className="navLink"
              href="/#section-faq"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              <div>
                Faq
              </div>
            </a>
          </div>
          <div
            className="nav-item"
          >
            <a
              className="navLink nav-link"
              data-rb-event-key="/#section-contact"
              href="/#section-contact"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              <div>
                Contact
              </div>
            </a>
          </div>
        </div>
      </nav>
    </div>
  </div>
</header>
`;
